File: NumberTranslatorStaticClassesApi/DbTables/NumberPrefixTable.cs
using System.Collections.Generic;
using NumberTranslatorStaticClassesApi.Models;

namespace NumberTranslatorStaticClassesApi.DbTables
{
    public static class NumberPrefixTable
    {
        public static List<NumberPrefix> Table => new()
        {
            new NumberPrefix { Number = 2, Text = "twen" },
            new NumberPrefix { Number = 3, Text = "thir" },
            new NumberPrefix { Number = 4, Text = "for" },
            new NumberPrefix { Number = 5, Text = "fif" },
            new NumberPrefix { Number = 8, Text = "eigh" }
        };
    }
}

File: NumberTranslatorApi/DbTables/NumberPrefixTable.fs
module NumberPrefixTable

type NumberPrefix = {
    Number: int
    Text: string
}

let table = 
    [|  
        {Number = 2; Text = "twen"}
        {Number = 3; Text = "thir"}
        {Number = 4; Text = "for"}
        {Number = 5; Text = "fif"}
        {Number = 8; Text = "eigh"}
    |]